bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -197,7 +197,7 @@ spec:
           gateway_mode_flags=
           # Check to see if ovs is provided by the node. This is only for upgrade from 4.5->4.6 or
           # openshift-sdn to ovn-kube conversion
-          if grep -q OVNKubernetes /etc/systemd/system/ovs-configuration.service ; then
+          if grep -q OVNKubernetes /etc/systemd/system/ovs-configuration.service && [ -f /host/var/run/ovs-config-executed ]; then
```

Comment:
first half of this is unnecessary now isn't it?

Reply:
it's still necessary for SDN conversion I think